content/en/docs/setup/learning-environment/kind.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Installing Kubernetes with Kind
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Kind is a tool for running local Kubernetes clusters using Docker container "nodes".
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Installation
+
+See [Installing Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+{{% /capture %}}
+
+
```

Comment:
How about a _What's next_ hyperlink to https://kind.sigs.k8s.io/docs/user/quick-start/#interacting-with-your-cluster?

Reply:
Also, please consider adding the weight: 20 in the front matter of the .md file.

Reply:
sure